traffic_ops/testing/api_contract/v4/response_template.json

```diff
@@ -176,6 +176,23 @@
     "server_capabilities": {
         "type": "object",
         "properties": {
+            "id": {
+                "type": "int"
+            },
```

Comment:
Not sure why you changed anything here, but it shouldnt have an id property. Tests are failing because of this

Reply:
I think it was a mistake in resolving the merge conflict.  I'll fix it.